src/assets/custom/css/_sass/_in-page-banner.scss

```diff
@@ -21,13 +28,20 @@
   align-items: center;
   background-color: $coral;
   display: flex;
-  height: 400px;
+  height: 300px;
```

Comment:
This number comes up a lot, is it worth putting it into an explainer variable?

Reply:
This is amended in the following commit:

[0dc1108](https://github.com/codurance/site/commit/0dc1108de9e7d03e9faefff61613d7f7b1e2ed2b)